hassio/secrets.py

```diff
@@ -55,7 +57,9 @@ def get(self, secret: str) -> Union[bool, float, int, str]:
 
             # Filter to only get supported values
             self.secrets = {
-                k: v for k, v in data.items() if isinstance(v, (bool, float, int, str))
+                k: v
+                for k, v in data.items()
+                if isinstance(v, (bool, float, int, str, list))
```

Comment:
The contents of lists are not filtered/checked this way...

Reply:
Since we support complex types and use schemes validation, we don't need the type check here?